scripts/etc/init.d/rcS

```diff
@@ -409,6 +413,15 @@ case "$rootfstype" in
     ;;
 esac
 
+# check if we need wpasupplicant package
```

Comment:
the comment is a bit misleading, as this isn't really a check if the packages is needed, but if an interface other than `eth0` is used 😉